README.md

```diff
@@ -693,17 +742,54 @@ Define the target field name used to retrieve the label of the referenced elemen
                 .targetField(nga.field('title')) // Select a label Field
         ]);
         
-* `singleApiCall(function(entityIds) {}`
+* `singleApiCall(function(entityIds) {})`
+
 Define a function that returns parameters for filtering API calls. You can use it if you API support filter for multiple values.
 
-		// Will call /posts?post_id[]=1&post_id[]=2&post_id%[]=5...
+		// Will call /posts?_filters=id1,id2,.._
```

Comment:
In fact, the URL generated by the singleApiCall() has always been by default 

    /posts?_filters=id1,id2,..

But the documentation didn't mention the fact that to get an url as

    /posts?post_id[]=1&post_id[]=2&...

you have to configure a Restangular interceptor.